src/backend/utils/wiki-feed-parser.js

```diff
@@ -11,7 +11,9 @@ const { JSDOM } = jsdom;
  * That data is then returned as a Promise
  */
 module.exports.getData = function() {
-  return fetch(process.env.FEED_URL)
+  return fetch(
+    process.env.FEED_URL || 'https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List'
```

Comment:
This is probably wise, since this URL doesn't change very often (if ever).  But could you add a log that says we're doing this? Something like:

```js
let ur = process.env.FEED_URL;
if(!url) {
  url = 'https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List'
  logger.debug('No value found for FEED_URL in env, using default ${url} instead');
}
```

This will help us debug things later, if we can't figure out why it's always loading a URL we don't intend.

Reply:
Great idea! Done: https://github.com/Seneca-CDOT/telescope/pull/550/commits/e4c5a12a32b21a76efdeaf8feede880ddb3f31d6